lib/uinta/formatter/util.ex

```diff
@@ -0,0 +1,82 @@
+defmodule Uinta.Formatter.Util do
```

Comment:
If we do create a separate formatter, I would probably tend to leave these in `Uinta.Formatter` (as well as the types) and not create a new "Standard" formatter.